include/ttmlir/Dialect/TTNN/Pipelines/Passes.h

```diff
@@ -87,6 +87,13 @@ struct TTIRToTTNNBackendPipelineOptions
           *this, "override-grid-sizes",
           llvm::cl::desc("Override grid sizes for specific ops."),
           llvm::cl::init(llvm::StringMap<SmallVector<int64_t, 2>>())};
+
+  // Option to provide a system descriptor flatbuffer file to compile against
+  Option<std::string> systemDescPath{
+      *this, "path",
```

Comment:
Use something more descriptive than "path". Why not "system-desc-path"?

Reply:
I initially changed it to system-desc-path but the feedback was it might break already set workflows of others (and its a sub-option of SystemDescPath already so seems redundant)

Reply:
I saw the first comment now. :) Still this is in TTNN pipeline. Look how CL command looks like:
// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="path=%system_desc_path%"
backend pipeline path?

Reply:
Yeah, sorry @tapspatel, before you updated the existing pass arg which would have broken existing workflows and where the context is clear that it's referring to system desc.  Since this arg is now introduced at the pipeline argument something more descriptive like `system-desc-path` would be better.

Reply:
very nitty but I think keeping the same variable option name for both passes makes sense (and consistent for users) @nsmithtt lemme know what you think. I think it makes sense to just keep it all system-desc-path (or a variant of this)

Reply:
cool I shall change it thanks

Reply:
I think we should keep the original as just `path` to avoid breakage.  I don't think it's a big deal that they mismatch, it's just extra churn if it breaks people